03.api.md

```diff
@@ -1099,3 +1098,228 @@ handler (const jerry_api_object_t * function_obj_p,
+- [jerry_api_set_object_native_handle](#jerryapisetobjectnativehandle)
+
+# jerry_api_set_array_index_value
+
+**Summary**
+Set value of a indexed element in the specified array object.
+
+**Prototype**
+
+{% highlight cpp %}
+bool
+jerry_api_set_array_index_value (jerry_api_object_t * array_object_p,
+                                 jerry_api_length_t index,
+                                 jerry_api_value_t * value_p);
+{% endhighlight %}
+
+- `array_object_p` - pointer to the array object;
+- `index` - index of the array element;
+- `value_p` - indexed value to set;
+- returned value - true, if value was set successfully.
+
+**Example**
+
+{% highlight cpp %}
+{
+    jerry_api_object_t * array_object_p = jerry_api_create_array_object (10);
+    jerry_api_value_t val;
+
+    ... // receive or construct val
+
+    jerry_api_set_array_index_value (array_object_p, 5, &val);
+
+    jerry_api_release_object (array_object_p);
+}
+{% endhighlight %}
+
+**See also**
+
+- [jerry_api_value_t](#jerryapivaluet)
+- [jerry_api_create_array_object](#jerryapicreatearrayobject)
+
+# jerry_api_get_array_index_value
+
+**Summary**
+Get value of a indexed element in the specified array object.
+
+**Prototype**
+
+{% highlight cpp %}
+bool
+jerry_api_get_array_index_value (jerry_api_object_t * array_object_p,
+                                 jerry_api_length_t index,
+                                 jerry_api_value_t * value_p);
+{% endhighlight %}
+
+- `array_object_p` - pointer to the array object;
+- `index` - index of the array element;
+- `value_p` - retrieved indexed value (output parameter);
+- returned value - true, if value was retrieved successfully.
+
+**Example**
+
+{% highlight cpp %}
+{
+    jerry_api_object_t* array_object_p;
+    ... // receive or construct array_object_p
+
+    jerry_api_value_t val;
+    bool is_ok = jerry_api_get_array_index_value (array_object_p, 5, &val);
+    if (is_ok)
+    {
+        ... // usage of 'val'
+    }
+}
+{% endhighlight %}
+
+**See also**
+
+- [jerry_api_value_t](#jerryapivaluet)
+- [jerry_api_create_array_object](#jerryapicreatearrayobject)
+
+# jerry_api_release_value
+
+**Summary**
+Release specified pointer to the value.
+
+**Prototype**
+
+{% highlight cpp %}
+void
+jerry_api_release_value (jerry_api_value_t * value_p);
+{% endhighlight %}
+
+- `value_p` - pointer to the value.
+
+**Example**
+{% highlight cpp %}
+{
+    jerry_api_value_t val1;
+    jerry_api_value_t val2;
+
+    val1.type = JERRY_API_DATA_TYPE_OBJECT;
+    val1.v_object = jerry_api_create_object ();
+
+    val2.type = JERRY_API_DATA_TYPE_STRING;
+    val2.v_string = jerry_api_create_string ("abc");
+
+    ... // usage of val1
+
+    jerry_api_release_value (&val1);
+
+    ... // usage of val2
+
+    jerry_api_release_value (&val2);
+}
+{% endhighlight %}
+
+**See also**
+
+- [jerry_api_value_t](#jerryapivaluet)
+
+# jerry_api_create_error
+
+**Summary**
+Create new Javascript error object.
```

Comment:
Other parts in the document uses the 'JavaScript' casing, we should follow that.